static/js/impala/users.js

```diff
@@ -72,7 +72,7 @@ $(function() {
 
         var required = 'input[aria-required=true], select[aria-required=true]';
 
-        $form.live('change input', required, validate_required_fields);
+        $form.on('change input', required, validate_required_fields);
```

Comment:
This is pretty self-contained code so I think this doesn't need to be delegated.